code/index.html

```diff
@@ -2,21 +2,92 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>classic newspaper</title>
     <link rel="stylesheet" href="./style.css" />
+    <meta name="viewport" content="width=device, initial-scale=1.0">
     <!-- dont forget the meta tag for responsiveness -->
   </head>
+ 
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+    <header>        
+       <section class="header-container" >
+          <img src="divider-4715969.svg" alt="devider">
```

Comment:
I don't think you need to add something in the `alt` here. If you leave it empty a screen reader will just skip the image. I think that is good as it's only something beautiful visually